### DISCUSSION TITLE
new stock of Thrust upgrade bundle for Crazyflie 2.X

### DISCUSSION THREAD

**Discussion by dhamidou** [bitcraze/discussions#1276] [General]
Hi, I wanted to order this :  [Thrust upgrade bundle for Crazyflie 2.X](https://store.bitcraze.io/products/thrust-upgrade-bundle-for-crazyflie-2-x?_pos=13&_sid=c8ede637f&_ss=r&variant=50059002413405),  but I see it's out of stock. Could you tell me when the item will be available?
Thanks in advance

**Comment by ToveRumar** [comment]
Hi!
Thanks for your question!
Hopefully it will be available within two months.

**Reply by dhamidou** [reply]
ok, thanks you
